jerry-core/ecma/operations/ecma-conversion.cpp

```diff
@@ -102,22 +102,30 @@ ecma_op_same_value (ecma_value_t x, /**< ecma-value */
   {
     return false;
   }
-
-  if (is_x_undefined
-      || is_x_null)
+  else if (is_x_undefined
```

Comment:
I think that in the case, in multiline mode, as equal `is_x_` parts are closer, it is simpler to compare the condition parts, and so, simpler to read the code.